src/Util/ColumnParser.php

```diff
@@ -51,11 +51,14 @@ public function parseFields($arguments)
                 }
             }
 
+            $nullable = substr($type, -1) === '?';
+            $type = $nullable ? substr($type, 0, strlen($type) - 1) : $type;
```

Comment:
The ``substr($type, 0, strlen($type) - 1)`` could be replaced with only ``substr($type, 0, -1)``